transaction/core/src/ring_signature/rct_bulletproofs.rs

```diff
@@ -350,7 +387,41 @@ fn sign_with_balance_check<CSPRNG: RngCore + CryptoRng>(
     })
 }
 
+/// Toggles between old-style and new-style extended message
```

Comment:
@mfaulk @isis-mc lmk how you guys feel about this, Matt I think you already commented positively on switching to Merlin. To keep compat though we have to use the old version until block version = 3 lands.